test/fixtures/user/migration.js

```diff
@@ -0,0 +1,38 @@
+const githubUserInfo = require("../auth/githubUserInfo")();
+
+/**
+ * User info for GitHub auth response
+ * Multiple responses can be added to the array if required
+ *
+ * @return {Object}
+ */
+module.exports = () => {
+  return [
+    {
+      username: "ankur",
+      first_name: "Ankur",
+      last_name: "Narkhede",
+      yoe: 0,
+      img: "./img.png",
+      linkedin_id: "ankurnarkhede",
+      github_id: githubUserInfo[0].username,
+      github_display_name: githubUserInfo[0].displayName,
+      isMember: true,
+      phone: "1234567890",
+      email: "abc@gmail.com",
+      roles: {
+        member: true,
+      },
+      tokens: {
+        githubAccessToken: "githubAccessToken",
+      },
+      status: "active",
+      profileURL: "https://abcde.com",
+      picture: {
+        publicId: "profile/mtS4DhUvNYsKqI7oCWVB/aenklfhtjldc5ytei3ar",
+        url: "https://res.cloudinary.com/realdevsquad/image/upload/v1667685133/profile/mtS4DhUvNYsKqI7oCWVB/aenklfhtjldc5ytei3ar.jpg",
+      },
+      incompleteUserDetails: false,
+    },
+  ];
+};
```

Comment:
Nit: 
Why you creating new file for this fixture...I think you created because github_user_id is not present
You can add this daya on test/fixtures/user/user.js
and then call as 
```
const userData = require("../fixtures/user/user")();
 to call particular fixture =>userData[4]
```

Reply:
I've added there also, migration fixture is required for one of the tests.